src/read.c

```diff
@@ -4710,18 +4732,24 @@ static avifResult avifMetaFindAlphaItem(avifMeta * meta,
         result = avifMetaFindOrCreateItem(meta, newItemID, alphaItem); // Create new empty item.
     }
     if (result != AVIF_RESULT_OK) {
-        avifFree(alphaItemIndices);
+        avifFree(dimgIdxToAlphaItemIdx);
```

Comment:
(This is a pre-existing issue.) Should we add `*alphaItem = NULL;` here?

`result` may be set to an error code at either line 4718 or line 4732. If we go through line 4718, then we have not assigned a value to `*alphaItem`.

Reply:
It seems there is no impact of doing so in case of error.